src/services/SyncService.js

```diff
@@ -268,6 +268,7 @@ class SyncService {
 			this.autosave.clear()
 		} catch (e) {
 			logger.error('Failed to save document.', { error: e })
+			throw e
```

Comment:
I thought I commented that already, but i feel this might be a risky change if the save fails during other code parts calling this method. Can we safeguard that other callers catch this error then and handle it gracefully?

Reply:
If we can get this in and backported more quickly without the throw e change, I'm fine with postponing this one :grimacing:

Reply:
I had a look at all occasions of `.save()`, `.autosave()` and `.forceSave()`. Seems to be mostly places where throwing an error should be save. But maybe it's better to remove this potentially dangerous change from this PR that is supposed to be backported and move it to a seperate PR that doesn't get backported.

Reply:
Did this now.